src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -1132,6 +1132,9 @@ private function registerTranslatorConfiguration(array $config, ContainerBuilder
 
             $translator->replaceArgument(4, $options);
         }
+
+        // Register the enabled locales
+        $container->setParameter('translation.enabled_locales', $config['enabled_locales']);
```

Comment:
Can we inject instead of setting a parameter (which is something we try to avoid if possible).

Reply:
Yes, much better. Done.